src/quick-lint-js/fe/parse-class.cpp

```diff
@@ -1126,6 +1129,40 @@ void parser::parse_and_visit_class_or_interface_member(
       }
     }
 
+    void error_if_access_specifier_not_first_in_field() {
+      if (p->options_.typescript && modifiers.size()) {
+        check_if_access_specifier_preceeds_given_modifiers({
+          token_type::kw_static,
+          token_type::kw_readonly
+        });
+      }
+    }
+
+    void error_if_access_specifier_not_first_in_method() {
+      if (p->options_.typescript && modifiers.size()) {
+        check_if_access_specifier_preceeds_given_modifiers({
+          token_type::kw_static,
+          token_type::kw_async
+        });
+      }
+    }
+
+    void check_if_access_specifier_preceeds_given_modifiers(std::vector<token_type> tokens) {
```

Comment:
Must fix: `std::vector<token_type>` is expensive and its features are unnecessary. Use `span<const token_type>` (from `<quick-lint-js/port/span.h>`) instead.

Reply:
Fixed